sdks/python/apache_beam/runners/dataflow/dataflow_runner.py

```diff
@@ -117,13 +117,15 @@ class DataflowRunner(PipelineRunner):
   # TODO: Remove the apache_beam.pipeline dependency in CreatePTransformOverride
   from apache_beam.runners.dataflow.ptransform_overrides import CombineValuesPTransformOverride
   from apache_beam.runners.dataflow.ptransform_overrides import CreatePTransformOverride
-  from apache_beam.runners.dataflow.ptransform_overrides import ReadPTransformOverride
   from apache_beam.runners.dataflow.ptransform_overrides import JrhReadPTransformOverride
+  from apache_beam.runners.dataflow.ptransform_overrides import ReadPTransformOverride
+  from apache_beam.runners.dataflow.ptransform_overrides import NativeReadPTransformOverride
 
-  # Thesse overrides should be applied before the proto representation of the
+  # These overrides should be applied before the proto representation of the
   # graph is created.
   _PTRANSFORM_OVERRIDES = [
-      CombineValuesPTransformOverride()
+      CombineValuesPTransformOverride(),
```

Comment:
@ananvay we should look at how these interact with auto-opt-in at the service.

Reply:
+1. To get some context, what do these overrides do differently for JRH vs. UW?

Reply:
Looking at other parts, seems like this would be the same for both? And the only difference would be the JRH_PTRANSFORM_OVERRIDES section below (which basically is JrhReadPTransformOverride)? I'm not sure why that Read override is needed differently for JRH...

Reply:
I'm not sure that it will have any effect on the service side. The v1b3 Job graph that is produced is the same.

Reply:
Correct, this change does not change the existing behavior, but the concern is that the existing behavior might be sub-optimal.